README.md

```diff
@@ -1,11 +1,20 @@
 # Awesome_Books
 
+> This is the first collaborative project in the Javascript module. It is the implementation of "Objects" and the "class"
+
+
 ## Built With
 - HTML5
 - Javascript
 - Linters
 
 
+## Live Demo
+
+[Live Demo Link](https://uwadonat.github.io/Awesome_Books/)
```

Comment:
- [ ] Kindly update the live demo link to reflect the changes in this branch.

Reply:
the live demo is now reflecting the changes.